app/controllers/admins/kudos_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Admins
+  class KudosController < AdminsController
+    before_action :set_kudo, only: [:destroy]
```

Comment:
looks ok, but the majority of ruby on rails developers don't like using callbacks
https://www.visuality.pl/posts/should-i-use-active-record-callbacks

Reply:
Thank you for sending the link to the article, it's actually not the best solution in the long term, but only for a small project